src/ResourceManagement/ResourceManager/Microsoft.Azure.Management.Fluent.ResourceManager/Core/RestClient/RestClient.cs

```diff
@@ -41,6 +41,11 @@ public IReadOnlyCollection<DelegatingHandler> Handlers
         {
             get
             {
+                for (int i = handlers.ToArray().Length - 1; i >= 0; --i)
```

Comment:
foreach (var handler in handlers)
{
        handler.InnerHandler = new HttpClientHandler();
}